config/initializers/subscribers.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+if Spree.solidus_gem_version < Gem::Version.new('2.11.0')
```

Comment:
Am I correct in saying this check is here because this gem does not currently support solidus 2.11 ?

If that's the case, we should probably just make that explicit in the gemfile, and remove this check, as reporting is a required feature for this gem's certification. (Making that limitation explicit could be done outside of this PR)

Reply:
See my comment below. We do support Solidus 2.11, this is just a "polyfill" for adding a missing feature (the order_recalculated event) which was only introduced in Solidus 2.11